eth/gasprice/gasprice.go

```diff
@@ -116,6 +122,16 @@ func NewOracle(backend OracleBackend, params Config) *Oracle {
 		maxBlockHistory = 1
 		log.Warn("Sanitizing invalid gasprice oracle max block history", "provided", params.MaxBlockHistory, "updated", maxBlockHistory)
 	}
+	congestedThreshold := params.CongestedThreshold
+	if congestedThreshold < 0 {
+		congestedThreshold = 0
+		log.Warn("Sanitizing invalid gasprice oracle congested threshold", "provided", params.CongestedThreshold, "updated", congestedThreshold)
+	}
+	defaultBasePrice := params.DefaultBasePrice
+	if defaultBasePrice == nil || defaultBasePrice.Int64() <= 0 {
+		defaultBasePrice = DefaultBasePrice
```

Comment:
set as 1 wei? because it's a bit weird that defaultBasePrice.Int64() <= 0 then set defaultBasePrice as 0 (DefaultBasePrice).

Reply:
It's true it doesn't really make sense in that regard. It should just be a sanitation check and makes sure it's not nil or below 0. I changed the condition to `defaultBasePrice == nil || defaultBasePrice.Int64() < 0` which then makes sense to set it to 0.